src/plugins/lua/spamassassin.lua

```diff
@@ -23,9 +23,7 @@ local rspamd_expression = require "rspamd_expression"
 local rspamd_mempool = require "rspamd_mempool"
 local rspamd_trie = require "rspamd_trie"
 local util = require "rspamd_util"
-local _ = require "fun"
```

Comment:
I thought about it, though I suppose this change should be in a separate commit.